spec/factories/folders.rb

```diff
@@ -7,5 +7,11 @@
     after :create do |folder|
       folder.collaborators << folder.user
     end
+
+    trait :with_35_associated_signs do
```

Comment:
nit: it doesn't seem that useful to add this when we're only using it once...

Reply:
whoops okay I'll change it